<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Credit {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5462c3, stop: 1 #ba872c);
}

QPushButton {
	border-radius: 10px;
	font-size: 30px;
	line-height: 60px;
}

QPushButton::pressed {
	color: black;
}</string>
  </property>
  <widget class="QDoubleSpinBox" name="credit_sum_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_label">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>29</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общая сумма кредита</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_combo_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="currentText">
    <string>Аннуитетный</string>
   </property>
   <item>
    <property name="text">
     <string>Аннуитетный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Дифференцированный</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="interest_rate_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_interest_rate_label">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>29</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка, % годовых</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="credit_term_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);
border: none;
padding-right: 10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QLabel" name="credit_term_label">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>159</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок кредита/займа</string>
   </property>
  </widget>
  <widget class="QComboBox" name="term_combo_box">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>200</y>
     <width>109</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>6</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);
border:  none;
border-left: 1px solid rgb(100,100,100);</string>
   </property>
   <property name="currentText">
    <string>лет</string>
   </property>
   <property name="maxVisibleItems">
    <number>14</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>месяцы</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="credit_type_label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип</string>
   </property>
  </widget>
  <widget class="QLabel" name="montly_payment_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="montly_payment_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment_box">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_payout_box">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_payout_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>280</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
